scripts/trigger-jenkins-build.js

```diff
@@ -124,4 +148,33 @@ module.exports = (app) => {
       githubClient.repos.checkCollaborator({ owner, repo, username: commentAuthor }, triggerBuildWhenCollaborator)
     })
   })
+
+  app.on('pull_request.opened', function handlePullCreated (event, owner, repo) {
+    const { number, logger, pull_request } = event
+    const pullRequestAuthor = pull_request.user.login
+    const options = {
+      owner,
+      repo,
+      number,
+      logger
+    }
+
+    function replyToCollabWithBuildStarted (err, buildUrl) {
+      if (err) {
```

Comment:
This does not actually reply to the thread right?

Reply:
Why yes it does https://github.com/nodejs/node/pull/21045#issuecomment-393675826

Reply:
@joyeecheung is right -- it originally commented, but then I removed the functionality (it relied on faulty Jenkins logic)